backend/src/test/java/com/redhat/cloud/notifications/db/EventLogCleanerTest.java

```diff
@@ -0,0 +1,86 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.TestLifecycleManager;
+import com.redhat.cloud.notifications.models.Event;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.smallrye.mutiny.Uni;
+import io.smallrye.mutiny.helpers.test.UniAssertSubscriber;
+import org.hibernate.reactive.mutiny.Mutiny;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import static com.redhat.cloud.notifications.db.EventLogCleaner.EVENT_LOG_CLEANER_DELETE_AFTER_CONF_KEY;
+import static com.redhat.cloud.notifications.db.EventLogCleaner.now;
+
+@QuarkusTest
+@QuarkusTestResource(TestLifecycleManager.class)
+public class EventLogCleanerTest {
+
+    /*
+     * The Event#created field is automatically set because of the @PrePersist annotation in CreationTimestamped when
+     * an Event is persisted using a stateful session. @PrePersist does not work with a stateless session. We need to
```

Comment:
Isn't the Event created on a stateless session too?

Reply:
https://github.com/gwenneg/notifications-backend/blob/72339b14c99c1a2433c56bd9f48592e4c664fbb5/backend/src/main/java/com/redhat/cloud/notifications/db/ApplicationResources.java#L169

And i suppose that's the `@PrePersist`

Reply:
I'm not sure I get the question. Yes the `Event` is persisted with the stateless session. The only difference with a stateful session is the invocation of `Event#prePersist` which sets the `created` field automatically with the stateful session. That invocation doesn't happen with the stateless session.

Reply:
I was asking because I didn't remember any call to `setCreated` in the previous PR, but then I recall about that `prePersist` call :-)